Sources/Plasma/Apps/plClient/external/render_svg.py

```diff
@@ -56,6 +57,11 @@
 except ImportError as e:
 	print("Rendering SVG resources requires CairoSVG.  Exiting...")
 	exit(1)
+try:
+	from PIL import Image, ImageFilter
+except ImportError:
+	print("Rendering SVG resources requires Pillow.  Exiting...")
+	exit(1)
```

Comment:
FWIW, if you execute `pip install cairosvg`, PIL(low) is installed as a dependency automatically.

Reply:
Ah, good point. That was just mindless copy-paste. But in the spirit of “include what you use”, I’ll leave it here if you don’t mind (a future CairoSVG might stop requiring PIL).